core/src/main/scala/com/dwolla/tracing/TraceWeaveCapturingInputs.scala

```diff
@@ -3,8 +3,95 @@ package com.dwolla.tracing
 import cats._
 import cats.syntax.all._
 import cats.tagless.aop.Aspect.Weave
+import com.dwolla.tracing.syntax._
 import natchez.Trace
 
+object TraceWeaveCapturingInputs {
+  def apply[F[_] : Apply : Trace, Cod[_]]: Weave[F, ToTraceValue, Cod, *] ~> F =
+    new TraceWeaveCapturingInputs[F, Cod]
+}
+
+/**
+ * Use this `FunctionK` when you have an algebra in
+ * `Weave[F, ToTraceValue, Cod, *]` and you want each method call on
+ * the algebra to introduce a new child span, using the ambient
+ * `Trace[F]`. Each child span will be named using the algebra name and
+ * method name as captured in the `Instrumentation[F, A]`, and the
+ * parameters given to the method call will be attached to the span
+ * as attributes.
+ *
+ * The format of the attributes is controlled via the implementation
+ * of the `ToTraceValue` typeclass. There are provided implementations
+ * for `String`, `Int`, `Boolean`, and `Unit`, as well as `Option[A]`
+ * where `ToTraceValue[A]` exists. Other types that have `Show` or
+ * Circe `Encoder` instances will also be converted.
+ *
+ * If a parameter is sensitive, one way to ensure the sensitive value
```

Comment:
To be clear, sensitive here means confidential or restricted, right? Such as passwords or PII

Reply:
Reading further made that clear 🤦

Reply:
Yep, exactly